content/en/docs/migration/flux-v1-migration.md

```diff
@@ -19,6 +19,14 @@ design.
 This may at times mean that you have to make adjustments to the way your
 current cluster configuration is structured. If you are in this situation
 and need help, please refer to the [support page](/support/).
+
+One key change in Flux v2 is "server-side apply" that enforces strict
+validation of the manifests.  It is important to note that manifests are
+applied atomically to the cluster only if YAML validation passes. If one
```

Comment:
"only if YAML validation passes" seems a bit technically imprecise but I think this is the right amount of detail for this early in the guide. Should we try to mention this again later, is there a right time in the progression of the text? WDYT